bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/StaticReferenceRewriteTransformerFactory.java

```diff
@@ -225,13 +231,21 @@ private String handleMatchingPatternAttribute(Pattern pattern, String attrValue)
                     // prepend host
                     url = prependHostName(url);
                     m.appendReplacement(sb, Matcher.quoteReplacement(url));
+                    // Added check to determine whether the existing host has to be replaced
+                    if (BooleanUtils.isTrue(this.replaceHost)) {
+                        int index = attrValue.contains("://") ? attrValue.indexOf("://") + 1 : 0;
```

Comment:
what about url's in the form of script src="//assets.adobedtm.com/..." ? This won't be covered by the logic you created